### PR TITLE
remove tsapi::c namespace

### PR DESCRIPTION
**Pull request by dragon512**

### CODE REVIEW

File: include/ts/remap_version.h [hunk 1]
```diff
@@ -43,5 +43,5 @@
   } while (0)
 
 #ifndef ATS_BUILD
```
Comment (cmcfarlen) [inline include/ts/remap_version.h:45]: Could remove the ATS_BUILD lines as well

**Review by cmcfarlen — CHANGES_REQUESTED** [review]
ATS_BUILD is not needed.

File: include/ts/remap_version.h (hunk 1 above)
Comment (dragon512) [reply include/ts/remap_version.h:45]: I was keeping it as a placeholder for when I add the namespaces back

Comment (dragon512) [reply include/ts/remap_version.h:45]: Still I will remove that...

**Review by cmcfarlen — APPROVED** [review]

**Pull request merged**